app/controllers/media_controller.rb

```diff
@@ -0,0 +1,7 @@
+class MediaController < ApplicationController
+  def index
+    @books = Book.all.order(:upvotes).reverse.take(10)
+    @albums = Album.all.order(:upvotes).reverse.take(10)
+    @movies = Movie.all.order(:upvotes).reverse.take(10)
```

Comment:
You could consider using a constant here to hold the number of media you want to display. That way if it changes later on, you only need to update it in one place instead of three.